frontend/src/pages/article/article-page.tsx

```diff
@@ -48,7 +48,7 @@ const ArticlePage: React.FC = () => {
     { id: 5, name: 'Tech' },
   ];
 
-  const { text, title, author, coverUrl } = article ?? {};
+  const { text, title, author, coverUrl, publishedAt, genre } = article ?? {};
```

Comment:
Not sure why do you need nullish coalescing - as you've already checked that `article` exists

Reply:
May I leave a default value of empty {} to prevent the error?